playbooks/openshift/README.md

```diff
@@ -201,9 +201,15 @@ The roles of the files are:
 
 Once you have all of this configured, running the actual installation is simple.
```

Comment:
The directory structure documentation here could be updated to include instructions on what data needs to be there so that the initialization playbook and further playbooks work. This change adds additional data requirements in the form of SSH private keys etc.